### PR TITLE
upgrade jenkins-k8s-node to 1.1.0

### PR DESCRIPTION
**Pull request by pydevops**

### CODE REVIEW

**Comment by pydevops** [comment]
test this please

**Comment by pydevops** [comment]
test this please

**Comment by pydevops** [comment]
test this please

**Comment by pydevops** [comment]
@geojaz  Besides using tag 1.1.0 on jenkins slave. As the CloudSQL can only reuse the instance name after 1 week, I have randomized the instance name in the scripts.

**Comment by chrislovecnm** [comment]
@pydevops we need seperate PRs for this please.  How was this working w/o your updates??

File: Makefile [hunk 1]
```diff
@@ -19,25 +19,24 @@ SHELL := /usr/bin/env bash
 linting: check_shell check_python check_golang check_terraform check_docker check_base_files check_headers check_trailing_whitespace
 
 SQLNAME := demo-postgres-8
```
Comment (techgnosis) [inline Makefile]: Need to lose this line

Comment (chrislovecnm) [reply Makefile]:
```suggestion
```

Comment (chrislovecnm) [reply Makefile]: Love this new toy 😆

**Comment by pydevops** [comment]
test this please

**Pull request merged**